force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -121,7 +151,13 @@ public with sharing class VariablesDataTypesOperators {
 	 * @return The complete sentence as a single String.
 	 */
 	public static String createSentence(String noun, String verb, String endingPunctuation) {
-		return null; // Replace null with the variable you used to store the result
+		String first = noun;
+        String second = verb;
```

Comment:
Same here. The method arguments (input parameters) can be used directly in the String concatenation on line 159